doc/code/qml_drawer.rst

```diff
@@ -42,6 +42,12 @@ The following images are generated via the code:
     qml.drawer.use_style(style)
     fig, ax = qml.draw_mpl(circuit)(1.2345, 1.2345)
 
+Note that the shown ``pennylane`` style can be replicated exactly by having the
+Quicksand Bold font installed and refreshing the Matplotlib font cache (by
+deleting the fontlist file in the Matplotlib cache directory) — to find the
+cache directory, run ``matplotlib.get_cachedir()``. If this font is not
+available, the drawer will fall back on a default font.
+
 Currently Available Styles
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. |bws| image:: ../_static/style/black_white_style.png
```

Comment:
For me the pennylane style looks slightly different, seems the font of the gate names is not bold.
![image](https://github.com/PennyLaneAI/pennylane/assets/43949391/69a9abbc-37e0-464c-a839-0b2f4e873ead)

compare to what is displayed online:
![image](https://github.com/PennyLaneAI/pennylane/assets/43949391/1c62221a-2901-4547-b19d-d052dbfaf60b)

did we change this or could this be something not rendering properly for me? I have Quicksand installed and deleted the json file as explained

Reply:
@Qottmann Do you have Quicksand Bold? 😅 It looks like that might be the only difference?

Reply:
I installed this https://fonts.google.com/specimen/Quicksand

Reply:
It's definitely the Quicksand font you have there, but I think you probably don't have the bold version installed. :)
![image](https://github.com/PennyLaneAI/pennylane/assets/4939555/bfee3ec0-5484-434c-b518-129cea9ec9f3)
This worked/works fine in the current version of PL.